client/signup/steps/domains/index.jsx

```diff
@@ -526,6 +527,9 @@ class DomainsStep extends React.Component {
 			);
 		} else if ( this.props.stepSectionName ) {
 			backUrl = getStepUrl( this.props.flowName, this.props.stepName, undefined, getLocaleSlug() );
+		} else if ( 0 === this.props.positionInFlow && selectedSite ) {
+			backUrl = '/view/' + selectedSite.slug;
```

Comment:
Just a matter of taste:

`/view/${ selectedSite.slug }`

Feel free to ignore.

Reply:
Implemented in [cbb4f6f](https://github.com/Automattic/wp-calypso/pull/31235/commits/cbb4f6fa614e6d2a3c44b688d615cf630cbd5d6c)